test/mocks/adloaderStub.js

```diff
@@ -13,7 +13,9 @@ function createStub() {
     } else if (typeof args[3] === 'function') {
       args[3]();
     }
-    return document.createElement('script');
+    const script = document.createElement('script');
+    script.src = args[0];
+    return script;
```

Comment:
So I am trying to pinpoint the issue, made my own branch with your changes https://github.com/prebid/Prebid.js/pull/6446

This change specifically seems to be causing problems, and I am banging my head on my desk trying to find out why.

All these tests running together, calling the adLoader, and some manipulating the script variable returned may be causing something to happen.

But when this above snippet is not checked in, the only errors are the Bid Won test in adlook spec:

![image](https://user-images.githubusercontent.com/13972794/111735977-31357600-883a-11eb-8038-ec50422025e1.png)


But when this is checked in, it is always:

![image](https://user-images.githubusercontent.com/13972794/111735950-24188700-883a-11eb-8416-96a0256db41d.png)

Reply:
Amazing error message, I know

Reply:
What is the context for modifying this function? Not sure I 100% understand why the adloaderStub is being updated.

Reply:
For me it [seems impossible to get local testing to match CircleCI's output](https://github.com/prebid/Prebid.js/pull/6308#issuecomment-802009747) and so my only hope at the moment is stare at the code, make a change, submit it, wait 15->30mins for 'yes'/no' feedback and no details as the CircleCI output is completely unavailable to me (unless I give CircleCI full read/write access to all my public/private repos).

...it is desperation at this point, an alternative methodology would be welcomed hugely.

Reply:
Ah. Sorry. I didn't realize it was just for testing purposes.

Reply:
this is the test that fails on edge, right?   https://github.com/prebid/Prebid.js/blob/595fc0a08b5f97a319b81f8099394623218efe7c/test/spec/modules/userId_spec.js#L2190

Reply:
> (unless I give CircleCI full read/write access to all my public/private repos).

or do the pr from a different github login ?

Reply:
> > (unless I give CircleCI full read/write access to all my public/private repos).
> 
> or do the pr from a different github login ?

...or use a CI provider that does not force all contributors to do this? We could argue both ways, it does not help :)

The problem here is that the described testing in the project readme does not match the testing you actually use and blocks PR contributions. So even if I handed everything over to CircleCI you still are asking me to wait ~30 minutes just to get any feedback and nothing actionable other than "Script error" to work with.

What would be infinitely helpful is if someone describes *exactly* what to type/run/use to get matching results. I have access to Browserstack, I run the same commands I see in `.circleci/config.yml` but I get different results. This needs to be fixed either by sending me clue on what I am doing wrong or updating your "how to test your changes" documentation.

Lets start with, can you show me what you run on your own workstation that replicates CircleCI's output? With that, I can then figure out where I am going wrong or it may highlight deficiencies in the documentation, either way it seems to be the right next step.

Reply:
I have tried a few things here and it seems to be a race condition on the behavioral testing side with the stub and different analytics adapters touching the DOM.

Reply:
I am happy to soak time into this if someone can explain the process to get matching results on my local workstation to what is coughed up by CircleCI.